lib/bin/gpa-games-cli

```diff
@@ -57,14 +57,14 @@ class HexCliGameState(UiGameState):
         ui_player = ui_player.strip().lower()
         if (ui_player == 'w' or ui_player == 'white'
                 or ui_player
-                   == COLOR_SYMBOLS[color_to_player(COLORS["white"])].lower()):
-            return color_to_player(COLORS["white"])
+                   == COLOR_SYMBOLS[color_to_player(COLORS['white'])].lower()):
+            return color_to_player(COLORS['white'])
         elif (ui_player == 'b' or ui_player == 'black'
                 or ui_player
-                   == COLOR_SYMBOLS[color_to_player(COLORS["black"])].lower()):
-            return color_to_player(COLORS["black"])
+                   == COLOR_SYMBOLS[color_to_player(COLORS['black'])].lower()):
+            return color_to_player(COLORS['black'])
         else:
-            raise("Unrecognized player, \"{}\"".format(ui_player))
+            raise('Unrecognized player, \'{}\''.format(ui_player))
```

Comment:
Should this line be `raise('Unrecognized player, "{}"'.format(ui_player))`? Then the backslashes can be removed and I think the double quotes make more sense here as far English grammar goes.